### PR TITLE
Use an environment variable for rosbridge ip

### PR DESCRIPTION
**Pull request by alexemdesgagne**
I added an environment variable for the rosbridge ip into docker compose local and AUV8. (Octopus Telemetry)

### CODE REVIEW

File: docker-compose.auv8.yml [hunk 1]
```diff
@@ -263,6 +263,8 @@ services:
     image: ghcr.io/sonia-auv/octopus-telemetry:develop
     container_name: octopus_telemetry
     network_mode: host
+    environment:
+      - REACT_APP_BRIDGE_IP=192.168.0.31
```
Comment (supertoto29) [inline docker-compose.auv8.yml]: @alexemdesgagne Shouldn't we place this IP in the launch as an environnement variable?? The end goal for the docker-compose is to have one docker-compose file for the 3 launch files.

Comment (alexemdesgagne) [reply docker-compose.auv8.yml]: That's a good point ! I'll fix that today !

Comment (alexemdesgagne) [reply docker-compose.auv8.yml]: Done. I used the variable defined in the bash file. Tested on my local machine and working !

**Review by supertoto29 — DISMISSED** [review]
LGTM!!

**Review by Kam0ss — APPROVED** [review]
LGTM

**Pull request merged**